packages/cloudflare/src/api/kvCache.ts

```diff
@@ -62,6 +79,17 @@ class Cache implements IncrementalCache {
           };
         }
       }
+
+      const entryValue = entry?.value as CachedFetchValue | undefined;
```

Comment:
Why do we need `as CachedFetchValue` here? 
Couldn't this be changed upstream?

Reply:
I've upstreamed the type, however I still need the `as CachedFetchValue` since typescript is unable to narrow down the type (because of the generic `IsFetch`) 😖